src/main/java/hu/dungeonhunter/characters/champion/Champion.java

```diff
@@ -89,10 +119,17 @@ public void drinkAHealingPotion() {
         }
     }
 
-    public void championAttackInitiationCalculation() {
-        System.out.print("Champion initiation: " + getInitiative() + " + ");
+    public void attackInitiationCalculation() {
+        System.out.print("Champion initiation calculation: " + getInitiative() + " + ");
         int champInitRoll = Dice.rollDice(10, 1);
         System.out.println("Final Champion initiation: " + (getInitiative() + champInitRoll));
-        finalChampionInitiation = getInitiative() + champInitRoll;
+        finalInitiation = getInitiative() + champInitRoll;
+    }
+
+    public void accuracyCalculation() {
+        System.out.print("Champion accuracy calculation: " + getAccuracy() + " + ");
+        int AccuracyRoll = Dice.rollDice(100, 1);
```

Comment:
1-100 roll for accuracy? Someone must be drunk :D

Reply:
Base accuracy +1d100 if higher like enemy defense, the hit will do damage. Its ok.